buildscripts/run_arm64_tests_in_docker.sh

```diff
@@ -3,24 +3,50 @@ set -ex
 
 readonly grpc_java_dir="$(dirname "$(readlink -f "$0")")/.."
 
-# Build magic docker image that can run on x86 host, but looks like an ARM machine
-# from the inside (qemu-user-static is used for emulation)
-# Run "sudo apt install qemu-user-static binfmt-support" install the emulator
-# on the host machine.
-# Also see https://ownyourbits.com/2018/06/27/running-and-building-arm-docker-containers-in-x86/
-cp /usr/bin/qemu-arm-static "${grpc_java_dir}/buildscripts/grpc-java-linux-arm64-tests"
-docker build -t grpc-java-linux-arm64-tests "${grpc_java_dir}/buildscripts/grpc-java-linux-arm64-tests"
-
 if [[ -t 0 ]]; then
   DOCKER_ARGS="-it"
 else
   # The input device on kokoro is not a TTY, so -it does not work.
   DOCKER_ARGS=
 fi
 
+docker build -t grpc-java-artifacts-aarch64 "${grpc_java_dir}"/buildscripts/grpc-java-artifacts-aarch64
+
+# build aarch64 protoc artifacts via crosscompilation
+# the corresponding codegen tests will be run under and emulator
+# (thanks to the binfmt_misc registration, the emulator will be automatically
+# used for executing aarch64 binaries, so we can execute the codegen tests
+# even though we are in a x86_64 docker container)
+docker run $DOCKER_ARGS --rm=true -v "${grpc_java_dir}":/grpc-java -w /grpc-java \
+  --user "$(id -u):$(id -g)" \
+  -e "JAVA_OPTS=-Duser.home=/grpc-java/.current-user-home -Djava.util.prefs.userRoot=/grpc-java/.current-user-home/.java/.userPrefs" \
+  grpc-java-artifacts-aarch64 \
+  bash -c "LOCAL_MVN_TEMP=$(mktemp -d) SKIP_TESTS=true ARCH=aarch_64 buildscripts/kokoro/unix.sh"
```

Comment:
`LOCAL_MVN_TEMP=$(mktemp -d)` seems like it does nothing, as unix.sh does the same.